local/backend.go

```diff
@@ -42,7 +46,13 @@ func init() {
 }
 
 func service(ctx context.Context) (backend.Service, error) {
-	apiClient, err := client.NewClientWithOpts(client.FromEnv, client.WithAPIVersionNegotiation())
+	options := apicontext.CliOptions(ctx)
```

Comment:
I really hate this way of passing hidden parameters through context and pretend we have a neutral API.
I'd prefer we adopt a more exeplicit `switch contextType` to initialize the adequate backend and parse command flags

Reply:
yes, we discussed this with Anca earlier, decided to get a first step like this to check everything is working fine, as changing the backend init API will impact a lot of places. But we should be doing this now with separate commit (change signature of backend init() func to pass what's required there). 
Using the context like this will also not work nicely with https://github.com/docker/compose-cli/pull/1397